packages/graphql-server/src/resolvers/documents/types.graphql

```diff
@@ -0,0 +1,72 @@
+scalar JSON
+
+type DocumentType {
+  id: String
+  value: String
+}
+
+type EmbeddedDocumentModel {
+  documentType: [DocumentType]
+}
+
+type DocumentModel {
+  id: String
+  created: String
+  modified: String
+  associatedType: String
+  associatedId: String
+  typeId: String
+  name: String
+  _eTag: JSON
+  _links: JSON
+  _embedded: EmbeddedDocumentModel
+}
+
+type PagedResultDocumentModel {
+  _embedded: [DocumentModel!]
+  pageNumber: Int
+  pageSize: Int
+  pageCount: Int
+  totalCount: Int
+  _links: JSON
+}
+
+type EntityTag {
+  buffer: String
+  offset: Int
+  length: Int
+  value: String
+  hasValue: Boolean
+}
+
+type FileData {
+  base64: String
+}
+
+type Query {
+  GetDocuments(
+    pageSize: Int
+    pageNumber: Int
+    sortBy: String
+    embed: [String]
```

Comment:
This should be an `enum` type like this
https://github.com/reapit/foundations/pull/2455/files#diff-e6bfd44f89f7928415ab41f135fe56f4